whats_left.py

```diff
@@ -483,11 +484,17 @@ def remove_one_indent(s):
 if args.signature:
     print("\n# mismatching signatures (warnings)")
     for modname, mismatched in result["mismatched_items"].items():
-        for (item, rustpy_value, cpython_value) in mismatched:
-            if cpython_value == "ValueError('no signature found')":
-                continue # these items will never match
-
-            print(f"{item} {rustpy_value} != {cpython_value}")
+        for i, (item, rustpy_value, cpython_value) in enumerate(mismatched):
+            if cpython_value and cpython_value.startswith("ValueError("):
+                continue  # these items will never match
+            if rustpy_value is None or rustpy_value.startswith("ValueError("):
+                rustpy_value = f" {rustpy_value}"
+            print(f"{item}{rustpy_value}")
+            if cpython_value is None or cpython_value.startswith("ValueError("):
```

Comment:
```suggestion
            if cpython_value is None:
```
Will this be the same logic? because `ValueError` is already filtered in line 488